src/injected/visualization/svg-shape-factory.ts

```diff
@@ -96,8 +97,10 @@ export class SVGShapeFactory {
         text.setAttributeNS(null, 'fill', textConfig.fontColor);
         text.setAttributeNS(null, 'text-anchor', textConfig.textAnchor);
 
-        if (tabOrder != null) {
+        if (tabOrder != null && !innerText) {
             text.innerHTML = tabOrder.toString();
+        } else if (innerText) {
+            text.innerHTML = innerText;
```

Comment:
Would the following give the same behavior?
```
if (innerText != null) { // set to innerText }
else if (tabOrder != null) { // set to tabOrder }
```
But the way you have it could be more readable so I'm fine with either way